src/Module.zig

```diff
@@ -4867,6 +4918,7 @@ pub fn declareDeclDependency(mod: *Module, depender_index: Decl.Index, dependee_
 pub const ImportFileResult = struct {
     file: *File,
     is_new: bool,
+    is_file: bool,
```

Comment:
nit: add a comment what type of file is meant. On Linux there are at least
File type | Description
-- | --
Ordinary or regular files | Contain data of various content types such as text, script, image, videos, etc.
Directory files | Contain the name and address of other files.
Block or character special files | Represent device files such as hard drives, monitors, etc.
Link files | Point or mirror other files
Socket files | Provide inter-process communication
Named pipe files | Allow processes to send data to other processes or receive data from other processes.

Reply:
This field refers to whether it was an `@import` of an actual path rather than a package, but maybe it's clearer to flip the boolean to `is_package`?